docs/community/liaoliao666-react-query-kit.md

```diff
@@ -0,0 +1,77 @@
+---
+id: liaoliao666-react-query-kit
+title: React Query Kit
+---
+
+🕊️ A toolkit for ReactQuery that make ReactQuery hooks reusable and typesafe
+
+## what do you benefit from it
+
+- Make `queryKey` strongly related with `queryFn`
+- Manage `queryKey` in a type-safe way
+- Generate quickly a custom ReactQuery hook
+- Make `queryClient`'s operations more clearly associated with which custom ReactQuery hook
```

Comment:
not sure about this sentence, it doesn't make much sense grammatically. Maybe:

```suggestion
- Make `queryClient`'s operations clearly associated with custom ReactQuery hooks
```

?

Reply:
It's done. thz u so much for check this.